client/my-sites/plan-features-2023-grid/actions.tsx

```diff
@@ -202,15 +205,45 @@ const LoggedInPlansFeatureActionButton = ( {
 	if (
 		( availableForPurchase || isPlaceholder ) &&
 		currentSitePlanSlug &&
-		isMonthly( currentSitePlanSlug ) &&
+		! current &&
```

Comment:
Adding this predicate explicitly, despite not being needed. I think it's noisy when several consecutive `if` statements with early returns need to execute in order. One early return is ok, but multiple ones start to feel overdone.